datadog_checks_base/tests/test_time.py

```diff
@@ -0,0 +1,58 @@
+# (C) Datadog, Inc. 2020-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+from datetime import datetime
+
+import pytest
+from dateutil import tz
+from six import PY2
+
+from datadog_checks.base.utils.time import EPOCH, UTC, get_current_datetime, get_timestamp, normalize_datetime
+
+pytestmark = pytest.mark.time
+
+
+def test_timestamp_type():
+    assert isinstance(get_timestamp(), float)
+
+
+class TestNormalization:
+    def test_replace(self):
+        now = datetime.now()
+        assert now.tzinfo is None
+
+        assert normalize_datetime(now).tzinfo is UTC
+
+    def test_utc(self):
+        nyc = tz.gettz('America/New_York')
+        now = datetime.now(nyc)
+
+        normalized = normalize_datetime(now)
+
+        assert normalized is now
+        assert normalized.tzinfo is nyc
+
+
+class TestCurrentDatetime:
+    def test_default_utc(self):
```

Comment:
1. It seems we are testing both `get_timestamp` and `get_current_datetime` at the same time, it's maybe better to separate that.

2. `assert get_timestamp(now) - get_timestamp(dt) < 0.01` seems using `get_timestamp` for actual and expected value. I think using a mocked `datetime.now`, the assertion might be more reliable. Something like:

`assert get_timestamp(dt) == 123455`